implementations/rust/ockam/ockam_app_lib/src/incoming_services/state.rs

```diff
@@ -165,6 +185,9 @@ pub struct IncomingService {
     // this enrollment ticket is modified to avoid conflicts with 'default' name
     // the name of the project is re-set to 'project_id'
     enrollment_ticket: EnrollmentTicket,
+    // When the invitation is removed, the service is marked as removed
+    // to clean up the resources before removing the service from the list
+    removed: bool,
```

Comment:
This variable is a little bit misleading. I'd rename it to something more explicit, like `marked_to_be_removed`.